db/migrate/20200501044034_add_subsample_to_project.rb

```diff
@@ -0,0 +1,8 @@
+class AddSubsampleToProject < ActiveRecord::Migration[5.1]
+  def change
+    change_table :projects, bulk: true do |t|
```

Comment:
We should discuss this pattern on future web meeting. I think we should start moving away from constantly extending these base tables and move these to extra columns into association tables. Alternatively, invest on JSON fields for these attributes (for which we should probably consider updating the DB to a version that supports it (although managed Aurora does not support MySQL 8).